R/ResultFlagsIndependent.R

```diff
@@ -978,28 +978,29 @@ TADA_FlagCoordinates <- function(.data,
 #' be flagged as potential duplicates.
 #'
 #' @return The same input TADA dataframe with additional columns: a
-#'   TADA.ProbableDuplicate column indicating if there is evidence that two or
-#'   more results are duplicated, a TADA.DuplicateID column, containing a
+#'   TADA.ProbableBetweenOrgDuplicate column indicating if there is evidence that two or
+#'   more results are duplicated, a TADA.BetweenOrgDupGroupID column, containing a
```

Comment:
suggestion:

"TADA.MultipleOrgDuplicate column indicating if there is evidence that results are likely duplicated due to submission of the same dataset by two or more different organizations, a TADA.MultipleOrgDupGroupID column"